resize.js

```diff
@@ -103,7 +103,15 @@ function splitUrl(url) {
   return url.match(/^\/(.*)_(\d+)_(\d+)(_(\d+)x)?\.(.*)/);
 }
 
-function getImageParams(url) {
+function getImageParams(req) {
+  var url = req.url;
+  var queryParams = null;
+  var split = url.indexOf('?');
```

Comment:
Yeah I tried that one. I could only get it to "extract" the path and the query string. I still had to parse the path and query string myself...